src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java

```diff
@@ -465,6 +465,22 @@ public ColorRGB[] executeSimple(Object[][] params) {
 		}).description("Returns a RGB color from the given red, green and blue parameters.")
 			.examples("dye player's leggings rgb(120, 30, 45)")
 			.since("2.5");
+
+		Functions.registerFunction(new SimpleJavaFunction<String>("repeat", new Parameter[] {
+				new Parameter<>("string", DefaultClasses.STRING, true, null),
+				new Parameter<>("count", DefaultClasses.NUMBER, true, null)
+			}, DefaultClasses.STRING, true) {
+				@Override
+				public String[] executeSimple(Object[][] params) {
+					String string = (String) params[0][0];
+					Number count = (Number) params[1][0];
+					if (count.intValue() < 0) return new String[]{string};
+					return new String[] {string.repeat(count.intValue())};
```

Comment:
Isn't [String#repeat](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#repeat(int)) added in Java 11? From what i see in [build.gradle,](https://github.com/SkriptLang/Skript/blob/master/build.gradle#L194) Skript still compiles with and targets Java 8.

You'll need to copy code of repeat function from newer Java versions or another solution (the implementation of repeat is [a bit long,](https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/lang/String.java#L4428) from a quick search, answers to [this](https://stackoverflow.com/questions/1235179/simple-way-to-repeat-a-string) SO question has a bunch of one-liner alternatives, although there is probably a reason Java's built-in method is that long, i.e performance and making sure everything goes correct, checking parameters etc.)

Probably just use the answer from second most upvoted [answer](https://stackoverflow.com/a/6857936) for Java 8, which is `String.join("", Collections.nCopies(count.intValue(), string));`

Reply:
I just looked into you're correct about this. I figured it would be fine since it ran just fine while testing I'll look into doing that I'll update it rn